roles/network_plugin/weave/tasks/seed.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Weave seed | Set seed if first time
+  set_fact:
+    seed: '{% for host in groups["k8s-cluster"] %}{{ hostvars[host]["ansible_default_ipv4"]["macaddress"] }}{% if not loop.last %},{% endif %}{% endfor %}'
```

Comment:
it just needs to be a unique value? why not set it to something more permanent and identifable like inventory_hostname or inventory_hostname + ip?